strings/ic_tips.txt

```diff
@@ -0,0 +1,28 @@
+To crack the safe in the vault, use a stethoscope or three plastic explosives on it.
+Interacting with a windoor rather then bumping into it will keep it open, you can interact with it again to close it.
+You can spray a fire extinguisher, throw items or fire a gun while floating through space to change your direction. Simply fire opposite to where you want to go.
+Firesuits and winter coats offer mild protection from the cold, allowing you to spend longer periods of time near breaches and space than if wearing nothing at all.
+Glass shards can be welded to make glass, and metal rods can be welded to make metal. Ores can be welded too, but this takes a lot of fuel.
+If you need to drag multiple people either to safety or to space, bring a locker or crate over and stuff them all in before hauling them off.
+Resisting will allow you to resist out of handcuffs, being buckled to a chair or bed, out of locked lockers and more. Whenever you're stuck, try resisting!
+You can move an item out of the way by dragging it and then clicking on an adjacent tile with an empty hand.
+You can recolor certain items like jumpsuits and gloves in washing machines by also throwing in a crayon.
+Maintenance is full of equipment that is different every shift. Look around and see if anything is worth using.
+There are many places around the station to hide contraband. A few for starters: linen boxes, toilet cisterns, body bags. Experiment to find more!
+You can use a machine in the vault to deposit cash or rob Cargo's department funds.
+You can screwdriver any non-chemical grenade to shorten fuses from 5 seconds, to 3 seconds, to 1 second, to instant boom! Nobody ever expects this.
```

Comment:
Not true

Reply:
fml do I need to update them as well as cull them 😩 I'll just write them myself

Reply:
> And I still don't like it because that's what the soapstone is for and that's a system that is a lot more interesting than reading the same 12 tips (most of which are of dubious helpfulness, and also OOC) over and over again

i've literally never seen an actual tip in a soapstone, nor anything of any actual substance, i don't think that's a good argument against this

Reply:
great chest ahead

Reply:
Try jumping